start_docker.sh

```diff
@@ -8,7 +8,7 @@ while test $# -gt 0
 do
   case "$1" in
     -d|--develop-notebook-templates)
-      JUPYTER_DIR='templates'
+      JUPYTER_DIR='templates_ark'
```

Comment:
Looks pretty minor, but did you check that these show up as expected when starting the docker?

Reply:
It works 👍.